caboose.go

```diff
@@ -85,11 +85,12 @@ const DefaultLoggingInterval = 5 * time.Second
 const DefaultSaturnOrchestratorRequestTimeout = 30 * time.Second
 
 const DefaultSaturnBlockRequestTimeout = 19 * time.Second
-const DefaultSaturnCarRequestTimeout = 30 * time.Minute
+
+const DefaultSaturnCarRequestTimeout = 1 * time.Minute
```

Comment:
Max P95 for a successful CAR fetch over the last 24 hours in prod is 30 seconds:

https://protocollabs.grafana.net/d/bxJz6bJ4k/bifrost-gw-metrics?orgId=1&from=now-24h&to=now&editPanel=134

Reply:
does this mean that a large movie won't work anymore?


I don't think this is an acceptable limit.

Reply:
Hmmm, I see what you mean.

Reply:
@willscott Reverted this change.